lib/Cake/Test/Case/Error/ErrorHandlerTest.php

```diff
@@ -203,7 +203,7 @@ public function testHandleErrorLoggingTrace() {
 			$result[0]
 		);
 		$this->assertRegExp('/^Trace:/', $result[1]);
-		$this->assertRegExp('/^ErrorHandlerTest\:\:testHandleErrorLoggingTrace\(\)/', $result[2]);
+		$this->assertRegExp('/^ErrorHandlerTest\:\:testHandleErrorLoggingTrace\(\)/', $result[3]);
```

Comment:
There was an extra line in the trace message (presumably because the additional method was called), pushing the line with `ErrorHandlerTest::testHandleErrorLoggingTrace()` down a line.